go.mod

```diff
@@ -77,6 +76,13 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
```

Comment:
Has go mod tidy been applied ? 
I don't understand why it's even allowed by golang

Reply:
It was applied, and I didn't manually modify this file, so it's solely golang's job

Reply:
It feels like a regression of golang, In the earliest version it would have been directly added to direct dependencies not grouped separately .

Maybe they want allow grouping dependencies by concerns.